src/components/DatePicker/DatePickerTrigger.tsx

```diff
@@ -18,12 +18,18 @@ export const DatePickerTrigger = forwardRef<HTMLDivElement, DatePickerTriggerPro
             <div
                 className={merge([
                     'tw-absolute tw-top-2 tw-text-black-60 tw-right-8 tw-z-10',
-                    value === '' ? 'tw-right-4' : 'tw-right-8',
+                    value === '' || !isClearable ? 'tw-right-4' : 'tw-right-8',
```

Comment:
Should we remove the `tw-right-8` from line 20 since we are defining this position on line 21?